addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/wordlists/smalllist.txt

```diff
@@ -0,0 +1,835 @@
+AID
```

Comment:
What's the source of this file? Should be mentioned somewhere.

Reply:
Arjun's small wordlist :). where do I mention it? I can do that in the code where I use it. For now, I have just kept it, it would be used while initializing the scanner with default params.

Reply:
In the code is fine for now, but should probably be in the help with proper credits.